crowbar_framework/app/models/node_object.rb

```diff
@@ -170,6 +170,10 @@ def self.all
     self.find nil
   end
 
+  def self.admin_node
+    all.select { |n| n if n.admin? }.first
```

Comment:
Performance/Detect: Use `find` instead of `select.first`. (https://github.com/JuanitoFatas/fast-ruby#enumerabledetect-vs-enumerableselectfirst-code)